Add policies readme (#400)
netgroup-polito/CrownLabs: infrastructure/policy-enforcement/README.md

> For more information visit the [gatekeeper official repository](https://github.com/open-policy-agent/gatekeeper)

Reviewer: ```suggestion
For more information visit the [gatekeeper official repository](https://github.com/open-policy-agent/gatekeeper).
```

## HOW TO USE GATEKEEPER

> You can find more information at this [link](../../policies/README.md)

Reviewer: ```suggestion
The following pages contain some more [information](../../policies/README.md) and [examples](./examples).
```

> You can find some examples at this [link](./examples)

Reviewer: ```suggestion
```
